instruqt/delivering-as-an-appliance/06-completing-the-install/assignment.md

```diff
@@ -0,0 +1,219 @@
+box, the process starts with a warning and instructions for the user to
+validate the self-signed certificate the installer created.
+
+![Explaining the Initial Certificate to Your Customer](../assets/certificate-warning.png)
+
+The next step will ask your customer to configure the permanent certificate
+for the Admin Console. They can choose between a self-signed certificate or
+uploading one signed by a certificate authority. Replicated recommends that they use a
+signed certificate, this is something you may also want to suggest in your
+installation documentation.
+
+For the lab, though, we'll stick with the self-signed certificate. When you
+click the "Continue to Setup" button, you'll get a warning about the initial
+self-signed certificate. Assuming you accept the risk, you'll get a form that
+allows you to set up the SlackerNews Admin Console certificate. Keep it set on
+"self-signed" and click "Continue".
+
+![Configuring the Admin Console Certificate](../assets/certificate-configuration.png)
+
+You'll be asked to log into the Admin Console on the next page. The password
+for the Admin Console is the password you specified in the last step. It will
+be `[[ Instruqt-Var key="ADMIN_CONSOLE_PASSWORD" hostname="node" ]]` if you use the suggestion
+from the lab text or skipped ahead to this step.
+
+### Adding Nodes to the Cluster
+
+The first thing you'll see when you log in to the Admin Console is the Node
+Management screen. This is where you can add additional nodes to the
+cluster embedded with SlackerNews. Node management is the first step in the install
+process because not all applications can run on a single node.
+
+![Admin Console Node Managemet Interface](../assets/node-management.png)
+
+To simplify this lab, we're going to stick with a single node cluster. Click
+"Continue" to continue the installation on one node. SlackerNews is a fairly
+lightweight application and can run fairly easily on a single node.
+
+### Configuring Your Instance
+
+Next is the configuratiion screen that we defined earlier. Fill in the values
+for each entry.
+
+![Entering the SlackerNews Configuration in the Admin Console](../assets/providing-configuration.png)
+
+The first two sections are fairly straightforward:
+
+* **SlackerNews Domain**: `[[ Instruqt-Var key="SLACKERNEWS_DOMAIN" hostname="node" ]]`
+* **Admin Users**: `[[ Instruqt-Var key="ADMIN_EMAIL" hostname="node" ]]`
+* **Service Type**: NodePort
+* **Node Port**: `30443`
+* **Certificate Source**: Generate
+
+The next section presents us with a bit of a challenge. We don't have a Slack
+team to connect to, so any values we enter will be invalid. The values are
+required, though, so we need to enter something. Let's use the following placeholder
+values:
+
+* **Slack Client ID**: `notavalidid`
+* **Slack Client Secret**: `notavalidsecret`
+* **User OAuth Token**: `xoxp-notavalidtoken`
+* **Bot User Auth Token**: `xoxb-notavalidtoken`
+
```

Comment:
There was also a "Postgresql" field with a "Deploy Postgresql Database" checkbox (enabled by default). 

Did you intend for this to be included? If so, could include a shoutout to let people know they can just leave it checked